infra/aws-ecs/services.tf

```diff
@@ -0,0 +1,84 @@
+resource "aws_ecs_service" "core" {
+  name = "core"
+  cluster = "${aws_ecs_cluster.cluster.id}"
+  task_definition = "${aws_ecs_task_definition.core.arn}"
+  desired_count = 5
+
+  depends_on = [ "aws_cloudwatch_log_group.core" ]
+}
```

Comment:
I only added `core` since it is the only service with an image getting published to dockerhub.